src/TMU.cpp

```diff
@@ -1,6 +1,10 @@
 #include <TMU.hpp>
 
 namespace TMU {
+
+TMU::TMU(DEV::MAX31855 t[4]) : thermocouples{
+    t[0], t[1], t[2], t[3]} {}
```

Comment:
I know I wrote this for you, but it looks a little weird. Take a couple minutes to see if there's any better way to initialize an array in an initializer list